Arista/ConfigFiles/telegraf-ninja.conf

```diff
@@ -0,0 +1,30 @@
+# Configuration for influxdb server to send metrics to
+[[outputs.influxdb]]
+  ## The full HTTP or UDP endpoint URL for your InfluxDB instance.
+  ## Multiple urls can be specified as part of the same cluster,
+  ## this means that only ONE of the urls will be written to each interval.
+  # urls = ["udp://localhost:8089"] # UDP endpoint example
+  urls = ["http://planck:8086"] # required
+  ## The target database for metrics (telegraf will create it if not exists).
+  database = "qubit" # required
```

Comment:
use some other database since this database is going to go away or might have been deleted already
check with @Manoj321 on having a new database
provide the details in aid/3771

Reply:
@ashish667 If there are going to be lot of ninja/qube related  stats going forward, I can create a new database for you. So that all the measurements are logically grouped.